tests/ui_tests/app/page/common/header.py

```diff
@@ -42,3 +43,17 @@ class AuthorizedHeaderLocators(CommonHeaderLocators):
     failed_job_button = Locator(By.XPATH, "//button[@mattooltip='Show failed jobs']", "Header button for failed Jobs")
     help_button = Locator(By.XPATH, "//button[@adcm_test='help']", "Header button for help")
     account_button = Locator(By.XPATH, "//button[@adcm_test='account']", "Header button for account settings")
+    block = Locator(By.XPATH, "//div[contains(@class, 'mat-menu-content')]", "Header popup block")
```

Comment:
`block` name looks too broad.

Reply:
no. it will be like AuthorizedHeaderLocators.block

Reply:
`Header popup ...` confused me.

Reply:
ah, yes, you're right. changed to popup_block